agentlocal/agentlocal.go

```diff
@@ -49,8 +49,10 @@ func SetTransport(ctx context.Context, debug bool) {
 type NetworkInfo bool
 
 const (
-	RequestNetworkInfo      NetworkInfo = true
-	DoNotRequestNetworkInfo NetworkInfo = false
+	RequestNetworkInfo        NetworkInfo = true //nolint
+	DoNotRequestNetworkInfo   NetworkInfo = false
+	Localhost                             = "127.0.0.1"
```

Comment:
do we really need `Localhost`?

Reply:
This was requested by @ShashankSinha252 to change it to localhost and add const.

Reply:
ok